09 Environment/MergeProcedure.md

```diff
@@ -0,0 +1,19 @@
+# Mergeprocedure for pull request. 
+
+**Before you merge a pull request.** 
+
+Before you make a merge any pull request into the master branch you need to make sure that the pull requst has been reviewed. 
```

Comment:
Look at this line again "Before you make a merge any pull request" it does not make sense, either you are missing a comma or you have to rewrite it a little